src/ui/sass/modules/_ResultsHeader.scss

```diff
@@ -32,13 +32,20 @@ $results-header-universal-background: var(--yxt-color-brand-white) !default;
   --yxt-results-header-universal-background: #{$results-header-universal-background};
 }
 
-.yxt-ResultsHeader {
+.yxt-ResultsHeader
+{
+  // reset styling from yxt-Results-filters, a css class added for backwards compatibility
```

Comment:
I'm worried about keeping reset styling in the SDK.

So the problem is we want to keep a class but we also want new styling. Is it possible to add this reset styling to a selector `yxt-Results-filters.yxt-ResultsHeader--universal` instead of for any use of `yxt-ResultsHeader`? That way it is easier to know when we can remove this styling.

Reply:
sure I like this a lot

Reply:
oh hmm we actually don't need a reset